utils/Logger.py

```diff
@@ -27,7 +28,10 @@ class DebugLevel(IntEnum):
 
 
 class Logger:
-    IS_WINDOWS = platform == 'win32'
+    
+    # If windows override whether to convert ANSI codes in the output into win32 calls.
+    if platform == 'win32':
+        init(convert=True)
```

Comment:
Can you please move this to line 30 of main.py? `colorama.init()` is being called there too. Maybe send `convert=platform == 'win32` or `convert=platform == 'nt'`, maybe the later is more correct in fact, not sure.

Reply:
Also please confirm that it works by testing in cmd, PowerShell and the integrated IDE console, I don't have a Windows machine ready to test it.

Reply:
Apparentally convert isn't necessary. Removing init() from Logger.py causes colors to work only when Logger methods are called from main.py where init() is called aswell. Calling init() (w/o convert) makes all Logger methods work. Same thing with all three, cmd, PowerShell and integrated IDE of VSC. Doing another commit on this.